RLBotDiscordBot/cogs/admin.py

```diff
@@ -119,6 +158,7 @@ def check_perms(self, ctx):
         roles_names = [r.name for r in ctx.author.roles]
         if ctx.message.channel.name != 'discord-bots':
             return False
+            print("fuck")
```

Comment:
please, don't use this language in an official repo XD

Reply:
Classic TGD!

Reply:
I removed it, I put it there while annoyed when working on it :P

Reply:
ah ok, sorry. i just notided the outdated badge